src/abaqus/Adaptivity/VelocityAdaptiveMeshConstraint.py

```diff
@@ -172,12 +172,12 @@ def setValues(
     def setValuesInStep(
         self,
         stepName: str,
-        v1: Union[SymbolicConstant, float] = None,
-        v2: Union[SymbolicConstant, float] = None,
-        v3: Union[SymbolicConstant, float] = None,
-        vr1: Union[SymbolicConstant, float] = None,
-        vr2: Union[SymbolicConstant, float] = None,
-        vr3: Union[SymbolicConstant, float] = None,
+        v1: Union[SymbolicConstant, float, None] = None,
+        v2: Union[SymbolicConstant, float, None] = None,
+        v3: Union[SymbolicConstant, float, None] = None,
+        vr1: Union[SymbolicConstant, float, None] = None,
+        vr2: Union[SymbolicConstant, float, None] = None,
+        vr3: Union[SymbolicConstant, float, None] = None,
```

Comment:
Actually, it seems that these arguments should not be `None`, it is set to `None` just because of wrong inference about the default values back in the initial implementation. For example, here these arguments should be required arguments instead of optional arguments.

Reply:
I set the default values by inferring them from the docstrings in the initial implementation, however, the Abaqus documentation has two different sections `Required Arguments` and `Optional Arguments`, unfortunately, I didn't use this information back then, resulting in the wrong inference about the default values.

Reply:
I created a new repository [abaqus-docs-md](https://github.com/haiiliin/abaqus-docs-md) storing markdown files where this project is originally from, it is Crawled from Abaqus documentation, we might use these files to correct the default values.

Reply:
Right! Some of the default values may be find in the documentation.

Reply:
But there are some `Optional Arguments` that does not have the default explicitly in the documentation. Also, there are some `Required Arguments` that may receive a default value in some cases, like the `end1Edges`, `end1Edges` arguments in the method [seedEdgeByBias](https://help.3ds.com/2022/English/DSSIMULIA_Established/SIMACAEKERRefMap/simaker-c-partmgnpyc.htm?contextscope=all&id=2c0fec4842e7464aab5a4d32a4e965fe#simaker-partseededgebybiaspyc).

For theses cases, I think we may use `Ellipsis` (`...`) as the default, following [this recommendation](https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values:~:text=In%20stubs%20it%20may%20be%20useful%20to%20declare%20an%20argument%20as%20having%20a%20default%20without%20specifying%20the%20actual%20default%20value.%20For%20example%3A).

Reply:
Right, good suggestion. And I think there are tons of incorrect default values, we may need to write some scripts to parse the markdown files and correct them.

Reply:
> 

These methods can be quite complicated, it is not both `end1Edges` and `end2Edges`, but one or two of them are required. Other examples like the [Set](https://help.3ds.com/2022/English/DSSIMULIA_Established/SIMACAEKERRefMap/simaker-c-setpyc.htm?contextscope=all&id=ac5fb20b71bb432fae0dbc7bcb65d8cb) method and [Surface](https://help.3ds.com/2022/English/DSSIMULIA_Established/SIMACAEKERRefMap/simaker-c-surfacepyc.htm?contextscope=all&id=8762ced06b534497987c2d1a589c10ed) method also behave like this. In the programming, they should be treated as `Optional Arguments`.